lib/gollum/public/gollum/javascript/gollum.js.erb

```diff
@@ -658,6 +658,37 @@ $(document).ready(function() {
         }
       });
     }
+
+    if (navigator.clipboard) {
+      // Create copy to clipboard buttons for code-blocks
+      const codeBlocks = document.querySelectorAll("pre > code");
+
+      const copyToClipboard = async ({ target }) => {
+        const codeBlock = document.querySelector(`#${target.dataset.codeblock}`);
+        await navigator.clipboard.writeText(codeBlock.textContent);
+
+        target.classList.add('active');
+        // wait 3 secconds
```

Comment:
There's a typo in this comment, but also, the timeout seems to be set to 1 second, not 3. :)

Reply:
Fixed!